src/read_api/interceptors.clj

```diff
@@ -1,15 +1,18 @@
 (ns read-api.interceptors
+  (:require [io.pedestal.http :refer [json-response]])
   (use [read-api.database]))
 
 (defn get-book-id-from-path [context]
   (-> context :request :path-params :book-id))
 
 (defn response [status body & {:as headers}]
-  {:status status :body (get-book-id-from-path body)})
+  {:status status :body body :headers {"Content-Type" "application/json"}})
 
-(def ok       (partial response 200))
-(def created  (partial response 201))
-(def not-found (partial response 404))
+(def ok          (partial response 200))
```

Comment:
Hm, Clojure doesn't have any library/plugin for thoose actions? ofc I only thinking about it, nothing to change :D

Reply:
Not really. I.e it knows how to serve the status codes. But I want to assign the codes with the response so I don't have to write it all the time

Reply:
Okay